src/main/java/com/exasol/releaserobot/release/MavenReleaseMaker.java

```diff
@@ -0,0 +1,42 @@
+package com.exasol.releaserobot.release;
+
+import java.util.logging.Logger;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+import com.exasol.releaserobot.MavenPlatform;
+import com.exasol.releaserobot.report.ReleaseReport;
+
+/**
+ * This class is responsible for releases on Maven Central.
+ */
+public class MavenReleaseMaker implements ReleaseMaker {
+    private static final Logger LOGGER = Logger.getLogger(MavenReleaseMaker.class.getName());
+    private final MavenPlatform platform;
+    private final ReleaseReport releaseReport;
+
+    /**
+     * Create a new instance of {@link MavenReleaseMaker}.
+     * 
+     * @param platform      instance of {@link MavenPlatform}
+     * @param releaseReport instance of {@link ReleaseReport}
+     */
+    public MavenReleaseMaker(final MavenPlatform platform, final ReleaseReport releaseReport) {
+        this.platform = platform;
+        this.releaseReport = releaseReport;
+    }
+
+    @Override
+    public boolean makeRelease() {
```

Comment:
I think we should let the exception propagate here.

IMO opinion using a return value here when this fails is not such a good idea cause all clients of this method need to check for this value every time to see if something went wrong. With exceptions this is cleaner. (have in mind that a method being not able to fulfill its purpose is an exceptional case)

In any case, if you really want to return value, a rather than returning a boolean might be better to return the validation report.

Reply:
this method also looks quite similar to the `GitHubReleaseMaker.makeRelease()`, perhaps we can extract it and use a template method on it.

We can do it in a next pull request